### PR TITLE
feat(oui-field): add custom message support

### PR DESCRIPTION
**Pull request by Jisay**
Add support for [ui-validator](https://github.com/angular-ui/ui-validate). 
See #148.

### CODE REVIEW

File: packages/oui-field/src/field.html [hunk 1]
```diff
@@ -23,22 +23,10 @@
...
-                ng-bind="$ctrl.getErrorMessage('minlength')"></span>
-            <span ng-message="maxlength"
-                ng-bind="$ctrl.getErrorMessage('maxlength')"></span>
-            <span ng-message="min"
-                ng-bind="$ctrl.getErrorMessage('min')"></span>
-            <span ng-message="max"
-                ng-bind="$ctrl.getErrorMessage('max')"></span>
-            <span ng-message="pattern"
-                  ng-bind="$ctrl.getErrorMessage('pattern')"></span>
+            <span data-ng-repeat="errorKey in $ctrl.getMessagesOrder()"
```
Comment (FredericEspiau) [inline packages/oui-field/src/field.html]: `track by`

File: packages/oui-field/src/field.html [hunk 2]
```diff
@@ -23,22 +23,10 @@
...
+                  ng-message="{{ errorKey }}"
```
Comment (FredericEspiau) [inline packages/oui-field/src/field.html]: `::`

File: packages/oui-field/src/field.html [hunk 3]
```diff
@@ -23,22 +23,10 @@
         ng-if="$ctrl.isErrorVisible()">
         <span ng-messages="$ctrl.getFirstError()" role="alert">
```
Comment (Jisay) [inline packages/oui-field/src/field.html]: I don't understand very well what the meaning of this test :-/

Comment (FredericEspiau) [reply packages/oui-field/src/field.html]: What test ?

Comment (Jisay) [reply packages/oui-field/src/field.html]: Sorry, I mean the method `getFirstError`

Comment (FredericEspiau) [reply packages/oui-field/src/field.html]: Only display the first error message I guess

**Pull request merged**